api_core/google/api_core/bidi.py

```diff
@@ -698,7 +702,11 @@ def stop(self):
             if self._thread is not None:
                 # Resume the thread to wake it up in case it is sleeping.
                 self.resume()
-                self._thread.join()
+                # The daemonized thread may itself block, so don't wait
+                # for it longer than a second.
+                self._thread.join(1.0)
+                if self._thread.is_alive():
+                    _LOGGER.warning("Background thread did not exit.")
```

Comment:
Seems coverage is upset we never hit this warning under test
google/api_core/bidi.py                                       257      1     66      1    99%   709, 708->709

Reply:
I don't like adding `# pragma: NO COVER`, but this is the perfect case for it: mocking that out would be worse.